app/services/create_performance_json.service.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const moment = require('moment')
+const lodash = require('lodash')
+
+const filteredServices = require('./get_filtered_services.service')
+const performanceDataService = require('./get_performance_data.service')
+
+async function createPerformanceJson() {
+  const fromDate = moment.utc().subtract(7, 'days').startOf('day').format()
+  const toDate = moment().utc().startOf('day').format()
+
+  const services = await filteredServices.fetchAndFilterServices()
+  const performanceReport = await performanceDataService.getPerformanceReport(fromDate, toDate)
+
+  const countBySector = {}
+  const countByOrganisation = {}
+
+  services.forEach(service => {
+    if (service.sector) {
+      const sector = service.sector
+      lodash.set(countBySector, sector, countBySector[sector] + 1 || 1)
+    }
+    const name = service.serviceName
+    lodash.set(countByOrganisation, name, countByOrganisation[name] + 1 || 1)
+  })
+
+  return  {
+    dateUpdated: moment().format('D MMMM YYYY'),
+    numberOfPayments: (performanceReport.payments.count / 1000000).toPrecision(3) + ' million',
```

Comment:
I think we want to used `toFixed(1)` to give 1 decimal place as we currently do? I think this will give the number that is currently `£ 936.2 million` as 936, and will could do this when we get above a billion:
```
// note that exponential notation might be returned in some circumstances
console.log((1234.5).toPrecision(2)) // logs '1.2e+3'
```